master/getting-started/kubernetes/installation/index.md

```diff
@@ -27,4 +27,9 @@ for more details.
 
 - [Google Compute Engine (GCE)](gce)
 
-- [Azure](azure)
\ No newline at end of file
+- [Azure](azure)
+
+If you will be using {{site.prodname}} Application Layer Policy then you will need to install
+[Istio](https://istio.io/) and use {{site.prodname}} manifests that enable this feature. Refer
+to [Application Layer Policy](app-layer-policy) for more details.
```

Comment:
I would suggest squashing your commits and rebasing off the latest master to pull in the latest for this page.